tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -7336,6 +7339,49 @@ <h2>Show similarity to selected datapoint</h2>
           }
         },
 
+        plotInferenceEvolution: function() {
+          const container = Polymer.dom(this.$$('#inferenceEvolution'));
+          while (container.firstChild) {
+            container.removeChild(container.firstChild);
+          }
+          const example = this.selectedExampleAndInference;
+          if (
+            !example ||
+            !example.inferences ||
+            example.inferences.length < 2
+          ) {
+            return;
+          }
+          const inferences = example.inferences;
+          const isBinary = this.isBinaryClassification_(
+            this.modelType,
+            this.multiClass
+          );
+          const data = [];
+          for (let i = 0; i < inferences.length; i++) {
+            const inference = inferences[i];
+            for (let modelInd = 0; modelInd < inference.length; modelInd++) {
+              const modelInference = inference[modelInd];
+              for (
+                let labelInd = isBinary ? 1 : 0;
```

Comment:
this logic isn't correct. what this does is only show the line for the 2nd-highest scoring class, not the class with class index 1, because this list is ordered by class score, not class ID. you'll need to check each entry and only use the one with class ID == 1. i saw this by running the multi class demo and seeing that when the class 1 score was high, the line in the graph was low.

Reply:
Indeed, James, thanks for this catch!
For the same reason the `labelInd` being used as key in the data doesn't mean what I expected.